src/main/kotlin/com/devfloor/untitled/hashtag/application/HashtagService.kt

```diff
@@ -0,0 +1,30 @@
+package com.devfloor.untitled.hashtag.application
+
+import com.devfloor.untitled.hashtag.domain.Hashtag
+import com.devfloor.untitled.hashtag.domain.HashtagRepository
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class HashtagService(
+    private val repository: HashtagRepository
+) {
+    @Transactional(readOnly = true)
+    fun showAllByNames(hashtags: List<String>): List<Hashtag> {
+        return repository.findByNameIn(hashtags)
+    }
+
+    @Transactional
+    fun createHashtags(hashtags: List<String>): List<Hashtag> {
+        val hashtagList = mutableListOf<Hashtag>()
+        for (index in hashtags.indices) {
+            hashtagList.add(repository.save(Hashtag(hashtags[index])))
+        }
+        return hashtagList
+    }
```

Comment:
이 부분이 아마 슬랙에서 말씀하셨던, 깃이 꼬여서 반영이 안된 부분인 것 같아요 !!
아래와 같이 바꿀 수 있을거같아요
```suggestion
    @Transactional
    fun createHashtags(hashtags: List<String>): List<Hashtag> {
        return hashtags.map {
            repository.save(Hashtag(it))
        }
    }
```

Reply:
오호 넵!! 감사합니다!